TEI_documentation/openStemmata.xml

```diff
@@ -0,0 +1,349 @@
+                    <moduleRef key="nets"
+                        except="arc forest graph iNode leaf listForest node root tree triangle"/>
+                    <moduleRef key="textcrit"
+                        except="app lacunaEnd lacunaStart lem listApp rdg rdgGrp variantEncoding wit witDetail witEnd witStart"/>
+                    
+                    
+<!-- element -->
+                    
+                    <elementSpec ident="ptr" mode="change">
+                        <attList>
+                            <attDef ident="type" mode="change" usage="rec">
+                                <valList type="closed" mode="add" >
+                                    <valItem ident="digitised"/>
+                                    <valItem ident="description"/>
+                                </valList>
+                            </attDef>
+                            <attDef ident="target" mode="change" usage="rec">
+                                <datatype>
+                                    <dataRef key="teidata.text"/>
+                                </datatype>
+                            </attDef>
+                        </attList>
+                    </elementSpec>
+                    
+                    <elementSpec ident="bibl" mode="change">
+                        <attList>
+                            <attDef ident="type" mode="change" usage="req">
+                                <valList type="closed" mode="add" >
+                                    <valItem ident="book"/>
+                                    <valItem ident="article"/>
+                                </valList>
+                            </attDef>
+                        </attList>
+                    </elementSpec>
+                    
+                    <elementSpec ident="label" mode="change">
+                        <constraintSpec ident="descId" scheme="schematron">
+                            <constraint>
+                                <sch:rule context="tei:label">
+                                    <sch:assert test="@xml:id" role="error">Please add a @xml:id
+                                        attribute</sch:assert>
+                                    <sch:assert test="matches(@xml:id, 'n_.*')">L'id doit commencer par n_</sch:assert>
+                                </sch:rule>
+                            </constraint>
+                        </constraintSpec>
+                        <attList>
+                            <attDef ident="type" mode="change">
+                                <valList type="closed" mode="add" >
```

Comment:
Un seul type ?

Reply:
Shouldn't we include a second type, as an implied default value when the attribute is not used (`defaultVal`, I believe) ?